.changeset/config.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@3.0.0/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "bigcommerce/catalyst" }
+  ],
+  "commit": false,
+  "fixed": [],
+  "linked": [],
+  "access": "public",
+  "privatePackages": {
+    "version": true,
+    "tag": true
+  },
```

Comment:
More information: https://github.com/changesets/changesets/blob/main/docs/versioning-apps.md

TL;DR: This option tells Changesets that while package.json files with `"private": true` indeed should not be published to NPM, that we still want to version, tag, and generate changelogs for them. This is useful for apps such as `apps/core` and `packages/components`, which won't be published to NPM, but will still need semver and changelogs. 

Unfortunately, this is all or nothing, so this behavior will apply to any package.json with `"private": true`, which includes:
- `apps/docs`
- `packages/functional`

Reply:
✅